clang/lib/Sema/SemaAvailability.cpp

```diff
@@ -210,13 +222,16 @@ static bool ShouldDiagnoseAvailabilityInContext(
   return true;
 }
 
-static bool
-shouldDiagnoseAvailabilityByDefault(const ASTContext &Context,
-                                    const VersionTuple &DeploymentVersion,
-                                    const VersionTuple &DeclVersion) {
+static unsigned getAvailabilityDiagnosticKind(
```

Comment:
Is `unsigned` really the best type we have for diagnostic kinds?

Reply:
I am not sure if it is the best, but that's what `SemaBase::Diag` method takes :)